index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700&family=Roboto&display=swap" rel="stylesheet">
+    <title>Luis Valera's Portfolio</title>
+</head>
+<body>
+    <header class="header">
+        <button type="button" class="menu-button">
+             <img src="img/Union.png" alt="menu-image">
+        </button>
```

Comment:
- Kindly wrap your hamburger button with a `nav` tag for semantics reasons :+1: .